### ISSUE TITLE
Error when 'make html' with citation

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1177]
- Operation System: Windows XP pro SP3, win32, Chinese Simplified 
- Sphinx version: 1.2b1           ( Sphinx-1.2b1-py2.7.egg )
- Python version: 2.7.5
- Docutils version: 0.10 release  (Auto installed by sphinx egg)
- Jinja2 version: 2.7             (Auto installed by sphinx egg)

The paragraph below is copied from [docutils-0.10.tar.gz]/docutils-0.10/docs/user/rst/demo.txt (line 89~101)

The paragraph cannot 'make html' until I add a '|' at the beginning of the line ([CIT2002]_)...

Is it a bug?

---

```
#!rest

Paragraphs contain text and may contain inline markup: *emphasis*,
**strong emphasis**, ``inline literals``, standalone hyperlinks
(http://www.python.org), external hyperlinks (Python_), internal
cross-references (example_), external hyperlinks with embedded URIs
(`Python web site <http://www.python.org>`__), footnote references
(manually numbered [1]_, anonymous auto-numbered [#]_, labeled
auto-numbered [#label]_, or symbolic [*]_), citation references
([CIT2002]_), substitution references (|example|), and _`inline
hyperlink targets` (see Targets_ below for a reference back to here).
Character-level inline markup is also possible (although exceedingly
ugly!) in *re*\ ``Structured``\ *Text*.  Problems are indicated by
|problematic| text (generated by processing errors; this one is
intentional).

```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1177
- Originally reported by: Polo Yu
- Originally created at: 2013-05-28T12:44:43.937

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-05-28 11:26:40+00:00_

[Polo Yu](https://bitbucket.org/poloyu), thanks for reporting! Please attach the error log.

**Comment by shimizukawa** [comment]
_From Polo Yu on 2013-05-28 21:21:57+00:00_

ErrorLog: sphinx-err-s6d969.log

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-06-18 14:17:34+00:00_

It's a bug. I'll correct ASAP.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-06-19 11:21:56+00:00_

Fixed by #1193
